packages/design-system/src/components/web-components/index.ts

```diff
@@ -12,7 +12,7 @@ export * from '../analytics';
 export * from '../flags';
 export * from '../i18n';
 
-export * from './ds-alert';
-export * from './ds-badge';
-export * from './ds-button';
-export * from './ds-usa-banner';
+import './ds-alert';
```

Comment:
curious why we have to import here instead of exporting

Reply:
We aren't actually exporting anything from those modules, and we just want to import them to get their side effects (registering the custom elements)